app/views/email_authentications/passwords/_expired_reset_token.html.erb

```diff
@@ -0,0 +1,12 @@
+<body class="error-body">
```

Comment:
I don't think we should be adding a `body` tag here, we'll already have one from the layout.

Reply:
You are probably correct there. I should've made this a draft PR because I had planned to go over styling with Claudio today. The layout was causing some slight issues, though I don't think it was because of this body tag.